server/config.go

```diff
@@ -17,6 +17,12 @@ import (
 	"github.com/miekg/dns"
 )
 
+const (
+	SCacheCapacity = 10000
```

Comment:
They're used as the defaults in the flag bindings.  I could make the flags a method in server you could invoke, but reuse of exact flags seems unlikely.  Let me know.


> On Jan 16, 2015, at 2:58 AM, Miek Gieben <notifications@github.com> wrote:
> 
> In server/config.go:
> 
> > @@ -17,6 +17,12 @@ import (
> >  	"github.com/miekg/dns"
> >  )
> >  
> > +const (
> > +	SCacheCapacity = 10000
> Do these still need to be exported? I didn't care before, because everything was in the main namespace
> 
> —
> Reply to this email directly or view it on GitHub.
>